Description: fix a FTBFS with GCC-5. Thanks to Linn Crosetto <linn@hp.com> for
             the first fix (see #777938). This patch closes #777966.
Author: Joao Eriberto Mota Filho <eriberto@debian.org>
Last-Update: 2015-07-02
Index: libpff-20120802/libuna/Makefile.am
===================================================================
--- libpff-20120802.orig/libuna/Makefile.am
+++ libpff-20120802/libuna/Makefile.am
@@ -4,7 +4,7 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir)/common \
 	@LIBCSTRING_CPPFLAGS@ \
 	@LIBCERROR_CPPFLAGS@ \
-	@MEMWATCH_CPPFLAGS@
+	@MEMWATCH_CPPFLAGS@ -std=gnu89
 
 noinst_LTLIBRARIES = libuna.la
 
